tests/shape_test.py

```diff
@@ -22,6 +23,42 @@ def test_fold():
     assert_export(sc.fold, x=da, dims=['x', 'y'], shape=[2, 3])
 
 
+def test_fold_size_minus_1():
+    x = sc.array(dims=['x'], values=np.arange(6.0))
+    da = sc.DataArray(x)
+    assert sc.identical(sc.fold(x, dim='x', sizes={
+        'x': 2,
+        'y': 3
+    }), sc.fold(x, dim='x', sizes={
+        'x': 2,
+        'y': -1
+    }))
```

Comment:
Sometimes I feel there must be a better way of configuring where our tooling adds linebreaks.